mne/io/brainvision/brainvision.py

```diff
@@ -195,7 +195,7 @@ def _read_vmrk_events(fname, event_id=None, response_trig_shift=0):
     for info in items:
         mtype, mdesc, onset, duration = info.split(',')[:4]
         onset = int(onset)
-        duration = int(duration)
+        duration = (int(duration) if duration.isdigit() else 0)
```

Comment:
wouldn't you want the default to be 1 and not zero, otherwise that marker won't be added?